example/example_options.py

```diff
@@ -0,0 +1,83 @@
+from schwab_api import Schwab
+from dotenv import load_dotenv
+import os
+import pandas as pd
+import pprint
+
+load_dotenv()
+
+username = os.getenv("SCHWAB_USERNAME")
+password = os.getenv("SCHWAB_PASSWORD")
+totp_secret = os.getenv("SCHWAB_TOTP")
+
+# Initialize our schwab instance
+api = Schwab()
+
+# Login using playwright
+print("Logging into Schwab")
+logged_in = api.login(
+    username=username,
+    password=password,
+    totp_secret=totp_secret # Get this using itsjafer.com/#/schwab.
+)
+
+#get quotes for options.
+OptionChain = api.get_options_chains_v2('$RUT') #try also with parameter greeks = True 
```

Comment:
nit: could we call this `option_chain`? `OptionChain` implies it is a class

Reply:
No pb. Will do